src/index.ts

```diff
@@ -48,6 +52,11 @@ program.command('repl')
 program.command('init')
   .description('Create a new Wollok project')
   .option('-p, --project [filter]', 'path to project', process.cwd())
+  .option('-n, --name [name]', 'name of the example', undefined)
```

Comment:
```suggestion
  .option('-n, --name [name]', 'name of the project', undefined)
```

O a qué te referís con example?

Reply:
Es que el project tiene un path, es raro que le pongas otro nombre...